frontend/packages/kubevirt-plugin/src/components/modals/description-modal/index.ts

```diff
@@ -0,0 +1 @@
+export * from './description-modal';
```

Comment:
lets avoid creating new index files, they cause cyclic deps and we will be removing them in the future